### PR TITLE
[#150] Removing irrelevant sections from my-document

### PR DESCRIPTION
**Pull request by philtuskha**
Besides commenting out UI code here, two things to note:
- new simple_tag to pluralize titles
- Changed verbose name of General Resource to `Document`

### CODE REVIEW

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -603,7 +604,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                     </a>
                                 </li>
                             {% endif %}
-                            {% if extended_metadata_exists or metadata_form %}
+                            <!-- {% if extended_metadata_exists or metadata_form %}
```
Comment (cbcunc) [inline hs_core/templates/pages/genericresource.html]: @philtuskha What is being hidden here?

File: hs_core/templates/pages/genericresource.html [hunk 2]
```diff
@@ -1053,7 +1054,7 @@ <h4>Contributors</h4>
                                 </div>
                             {% endif %}
                             <!-- ========= RESOURCE SPECIFIC TAB ========= -->
-                            {% if site or extended_metadata_exists or res_add_metadata or metadata_form %}
+                           <!--  {% if site or extended_metadata_exists or res_add_metadata or metadata_form %}
```
Comment (cbcunc) [inline hs_core/templates/pages/genericresource.html]: @philtuskha And what is being hidden here?

**Review by dsummersl — CHANGES_REQUESTED** [review]
Looks so much better!

Couple issues I noticed:
 * Related resources is still visible.
 * There is an extra text field visible in edit mode. (I'll attach a screenshot).

**Comment by dsummersl** [comment]
extra-text-field

**Comment by philtuskha** [comment]
@dsummersl 
1) Chris and I discussed leaving Related resources (in slack) due to that being the only place where you can reference the original to you current copy of your document.  😭   

2) It looks like that extra text field is the way the UI looks from Hydroshare. 
 I can maybe look into how to hide until there is content in that text box?  Here is the behavior when you add a keyword: http://recordit.co/HKFci9ZISQ

**Comment by dsummersl** [comment]
1. gotcha.
2. Ah...I didn't quite realize that. Can we put a placeholder text in there when it is empty maybe?

**Comment by philtuskha** [comment]
placeholder text ✔️

File: hs_core/templates/pages/genericresource.html (hunk 2 above)
Comment (philtuskha) [reply hs_core/templates/pages/genericresource.html]: THis? 
image

I frankly don't know what this is or what it is used for.  It didn't seem relevant to MYHPOM, however.  So I hid it.

I sluethed around a little more and figured out why this tab started to appear without any content and I think I have a less intrusive change.    

I'll push the change to the branch next.

**Comment by philtuskha** [comment]
P.S. There actually wasn't an input, so no placeholde. There were some empty `<ul></ul>` tags.  I futzed with the javascript to give a better experience there.

**Review by dsummersl — APPROVED** [review]
looks good! :1st_place_medal:

**Pull request merged**